oxidation/libparsec/crates/client_types/src/local_device.rs

```diff
@@ -1,5 +1,7 @@
 // Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
 
+use std::path::{Path, PathBuf};
+
 use libparsec_serialization_format::parsec_data;
```

Comment:
`libparsec_serialization_format` must be with others `libparsec` import (`std` too)